client/src/search/Typeahead/Typeahead.js

```diff
@@ -0,0 +1,229 @@
+
+    const queried_results = _.filter(
+      all_options,
+      (res) => matches_query(res) && !_.isUndefined(res.config_group_index)
+    );
+
+    const menu_props = {
+      queried_results,
+      pagination_size,
+      search_text,
+      config_groups,
+      on_select_item: this.on_select_item,
+      hide_menu: this.hide_menu,
+    };
+
+    return (
+      <div className="rbt" style={{ position: "relative" }}>
+        <div className="search-bar">
+          <div className="search-icon-container">
+            <span aria-hidden="true">
+              <img
+                src={`${get_static_url("svg/search.svg")}`}
+                style={{ width: "30px", height: "30px" }}
+              />
+            </span>
+          </div>
+          {filter_content && (
+            <div style={{ order: 2 }}>
+              <DropdownMenu
+                dropdown_trigger_txt={
+                  <div
+                    style={{
+                      textAlign: "start",
+                      whiteSpace: "nowrap",
+                      display: "inline-block",
+                    }}
+                  >
+                    <MediaQuery minWidth={breakpoints.minSmallDevice}>
+                      <div
+                        style={{
+                          whiteSpace: "nowrap",
+                          display: "inline-block",
+                          marginRight: "1.5rem",
+                        }}
+                      >
+                        <IconFilter
+                          height="5px"
+                          width="5px"
+                          vertical_align="top"
+                          alternate_color="false"
+                        />
+                      </div>
+                    </MediaQuery>
+                    <span>
+                      {is_original_filter
+                        ? text_maker("add_filter")
+                        : text_maker("edit_filter")}
+                    </span>
+                  </div>
+                }
+                dropdown_a11y_txt={
+                  is_original_filter
+                    ? text_maker("add_filter")
+                    : text_maker("edit_filter")
+                }
+                opened_button_class_name={"btn-ib-light--reversed--with-icon"}
+                closed_button_class_name={"btn-ib-light--with-icon"}
+                dropdown_content_class_name="no-right"
+                dropdown_content={filter_content}
+              />
+            </div>
+          )}
+          <input
+            id="rbt-search"
+            style={{ width: "100%", order: 1 }}
+            placeholder={placeholder}
+            aria-label={placeholder}
+            onChange={update_search_text}
+            ref={this.typeaheadRef}
+            onKeyDown={this.handleKeyDown}
+            value={search_text}
+            onFocus={() => this.setState({ can_show_menu: true })}
+            onKeyDown={this.handle_key_down}
+          />
+        </div>
```

Comment:
the voiceover likes to overlap the content it reads....

"72 results are f-"(status text) "tre"(search text)

tried putting aria live to assertive but doesn't seem to do much. any tips?

Reply:
Huh, polite is supposed to wait and atomic is supposed to prevent it from being read in pieces. Nothing off the top of my head, give it a google. If you don't find anything let me know and I'll look in to it myself.

Reply:
I think it reads it, but the input aria is a lot more assertive? I've tried using aria-disable on it but to no avail. Rather than being read in pieces, it straight up stops...

I've found a post on SO similar to what I'm facing except its broken in chrome for me as well...
https://stackoverflow.com/questions/62882886/safari-voiceover-issue-when-reading-input-field